meetings/meetings/settings/defaults.py

```diff
@@ -176,6 +176,7 @@
 
 # Where users are redirected after login
 LOGIN_REDIRECT_URL = 'http://localhost:4200'
+ACCOUNT_LOGOUT_REDIRECT_URL = 'http://localhost:4200'
```

Comment:
The only change I would suggest here, is to leave these blank and set them in `local-dist.py`